operators/README.md

```diff
@@ -73,8 +75,15 @@ The Instance Operator implements the backend logic necessary to spawn new enviro
 ### Persistent Feature
 The Instance Operator allows you to create persistent Virtual Machines. Persistent means that the VM can be stopped and restarted, deleted and recreated without data loss. To make this possible, the content of the VM's disk is stored in a PVC.
 In order to use this feature, [the CDI operator](../infrastructure/virtualization/README.md) has to be installed.
-Granted the CDI operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which version to create is performed looking at the corresponding field in the `template` spec.
-N.B. The process of creating a persistent VirtualMachine can take a bit more time with the respect to a normal one (5-10 mins, depending on the size of the image). However when you recreate the VM you will not have to wait such that time.
+
+Granted the CDI Operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which version to create is performed looking at the corresponding field in the `template` spec. It is also possible to select the amount of memory needed for the VM by modifying the corresponding `template` spec.
```

Comment:
The last sentence is not clear to me. First, I suppose you are talking about disk size, not memory. Second, I would say that the disk is created of the size specified by the template (IMO the word *modifying* is misleading, as it is not clear that it is configured by the manager of the template and not by the operator).